cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -701,6 +701,25 @@ def _validate(self, queues, new_storage_count):
             )
 
 
+class UnmanagedFsxMultiAzValidator(Validator):
+    """
+    Unmanaged FSx Storage Vs Multiple Subnets validator.
+
+    Unmanaged FSx volumes can exist in AZ that are different from the ones defined in queues configuration.
+    In these cases we notify customers that they may incur in increased latency and costs.
+    """
+
+    def _validate(self, queues, fsx_subnets):
+        for queue in queues:
+            if not set(queue.networking.subnet_ids) == set(fsx_subnets):
```

Comment:
Are we sure ONTAP is multi-zonal in the sense that it supports different mount targets on different AZs? If not, would this be the right logic for a multizonal ONTAP?

Reply:
Additional comment (more important): this poses the "constraint" (although it's just an INFO) to use the same subnets between the queue and the FSx. We should check the AZs, not the subnets.

Reply:
I totally agree.
I fixed the check and now uses AZ list instead of subnet list.